scene/resources/gradient.cpp

```diff
@@ -86,7 +86,7 @@ Vector<Color> Gradient::get_colors() const {
 	return colors;
 }
 
-void Gradient::set_offsets(const Vector<float> &p_offsets) {
+void Gradient::set_offsets(const Vector<real_t> &p_offsets) {
 	points.resize(p_offsets.size());
 	for (int i = 0; i < points.size(); i++) {
 		points.write[i].offset = p_offsets[i];
```

Comment:
`points` is still holding `float` offsets, should that change?

Reply:
I'm not sure. I undid the changes to `Gradient` in the meantime.